src/App.jsx

```diff
@@ -14,6 +15,7 @@ function App() {
         <Route path="/" element={<Home />}/>
         <Route path="/articles" element={<AllArticles/>} />
         <Route path="/articles/:article_id" element={<SingleArticle/>} />
+        <Route path="/articles/:article_id/comments" element={<Comments/>} />
```

Comment:
I don't think this route is used anywhere - no links I can find lead to it. Do you need it?